eiffel/src/main/java/com/etiennelenhart/eiffel/viewmodel/EiffelViewModel.kt

```diff
@@ -111,3 +153,9 @@ abstract class EiffelViewModel<S : State, A : Action>(
         scope.cancel()
     }
 }
+
+internal fun <S : State, A : Action> EiffelViewModel<S, A>.log(message: String) {
+    if (Eiffel.debugConfig.enabled && debug) {
```

Comment:
If you agree with the changes from the comment in line 49, this should also be changed to `Eiffel.debugConfig.enabled && !excludeFromDebug`.